src/layouts/Overview/components/BalanceInfo/components/GetPolyx/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useContext, useState } from 'react';
+import { AccountContext } from '~/context/AccountContext';
+import { Modal } from '~/components';
+import { Button, Heading, Text } from '~/components/UiKit';
+import { StyledLink, ConsentRow, ConsentCheckbox } from './styles';
+
+interface GetPolyxProps {
+  toggleModal: () => void;
+}
+
+export const GetPolyx: React.FC<GetPolyxProps> = ({ toggleModal }) => {
```

Comment:
It's just an additional layer of defense in case we got malicious env var injected into our systems. 

```suggestion
const validateBanxaUrl = (url: string): boolean => {
  try {
    const parsedUrl = new URL(url);
    return parsedUrl.protocol === 'https:' && 
           parsedUrl.hostname.endsWith('banxa.com');
  } catch {
    return false;
  }
};

export const GetPolyx: React.FC<GetPolyxProps> = ({ toggleModal }) => {
```